simulation/src/test/java/com/palantir/dialogue/core/SimulationTest.java

```diff
@@ -393,6 +393,35 @@ public void uncommon_flakes() {
                 .run();
     }
 
+    /**
+     * This simulates an alta client, which might load up some keys and then lookup each key in order to build a big
+     * response for the user. The goal is 100% client-perceived success here, because building up half the response
+     * is no good.
+     */
+    @Test
+    public void one_big_spike() {
+        int capacity = 100;
+        servers = servers(
+                SimulationServer.builder()
+                        .serverName("node1")
+                        .simulation(simulation)
+                        .handler(h -> h.respond200UntilCapacity(429, capacity).responseTime(Duration.ofMillis(150)))
```

Comment:
Is 429 representative here? I think we use 503 in most cases, unless alta does something special.

Reply:
Looks like alta uses `QosException.throttle()` (see QosAltaTransaction.java#L186), so I think it is representative.

Reply:
👍 thanks. I'm not sure if it's intentional that ConcurrencyLimitedChannel only checks 429 and not 503.

Reply:
Yeah that just seemed weird to me too!